lib/rules/order-in-components.js

```diff
@@ -231,7 +232,9 @@ module.exports = {
     ],
     messages: {
       order:
-        'The "{{name}}" property should be above the "{{firstUnorderedPropertyName}}" property on line {{line}}.'
+        'The "{{name}}" property should be above the "{{firstUnorderedPropertyName}}" property on line {{line}}.',
+      hasSideEffect:
+        'There may be some side effects, the "{{name}}" property should be manually moved above the "{{firstUnorderedPropertyName}}" property on line {{line}}.'
```

Comment:
Please rename this message to `reorderWithSideEffects` and change its wording to `Manually move "{{name}}" property above "{{firstUnorderedPropertyName}}" property on line {{line}} (might break side effects).`